ansible_collections/arista/avd/roles/eos_validate_state/ANTA-Preview.md

```diff
@@ -151,6 +151,9 @@ title: Ansible Collection Role eos_validate_state - Preview Integration with ANT
 - (New) AvdTestAPIHttpsSSL (No Ansible tags, use the new `skipped_tests` variable instead)
   - VerifyAPIHttpsSSL: Validate eAPI HTTPS SSL profile status.
 
+- (New) AvdTestStun (No Ansible tags, use the new `skipped_tests` variable instead)
+  - VerifyStunClient: Validate the STUN client is configured with the specified IPv4 source address and port. Validate the public IP and port if provided. For now AVD use source port as static 4500 and do not validate public IP and port. Please add the test in custom anta catalogs to validate public ip and port.
```

Comment:
```suggestion
  - VerifyStunClient: Validate the STUN client is configured with the specified IPv4 source address and port. Validate the public IP and port if provided. For now AVD sets the source port to 4500 and does not validate public IP and port. Please add the test in a custom ANTA catalog to validate public IP and port.
```

Reply:
Remove the source port 4500 for now. Add stuff about the fact that we do STUN check only for WAN case for now (because we look for path-groups)